qi-lib/flow/core/compiler.rkt

```diff
@@ -3,497 +3,15 @@
 (provide (for-syntax compile-flow normalize-pass))
 
 (require (for-syntax racket/base
-                     syntax/parse
```

Comment:
Ah, was this (e23c813ab0889cb9a635d12872ccd86c434f056f) missing from 2247005 (Preliminary splitting of the compiler into
separate modules for separate passes., 2024-05-05)? It would be good to get it
squashed back into that.